Dockerfile

```diff
@@ -3,6 +3,7 @@ WORKDIR /go/src/github.com/openshift/cluster-logging-operator
 
 # COPY steps are in the reverse order of frequency of change
 COPY cmd ./cmd
+COPY internal ./internal
```

Comment:
will need to move to Dockerfile.in if  https://github.com/openshift/cluster-logging-operator/pull/992  lands first

Reply:
I don't find Dockerfile.in in this repo. let me know if its belong to other internal repo..

Reply:
yes #992 pulls that file in.  if you rebase on upstream/master you will be able to make the needed changes